tests/console/nested_vm.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Package: nest_vm
+# Summary: Basics libvirtd test
```

Comment:
Would you adding a bit more of the intention of the test:

e.g: ` Test libvirt pre and post migration, with a nested VM running or shutdown, performance of the nested vm is irrelevant as long as the service is able to complete the upgrade` or similar to what the schedule says :)